README.md

```diff
@@ -4,7 +4,14 @@
 ![Lint Code Base](https://github.com/sleighzy/ansible-zookeeper/workflows/Lint%20Code%20Base/badge.svg)
 ![Ansible Lint](https://github.com/sleighzy/ansible-zookeeper/workflows/Ansible%20Lint/badge.svg)
 
-Ansible role for installing and configuring Apache ZooKeeper on RedHat 7 and 8.
+Ansible role for installing and configuring Apache ZooKeeper
+
+| Platform / Status | Supported           | Tested             |
+| ------------------|:-------------------:|:------------------:|
+| Debian 10.x       | Yes                 | Yes                |
+| Ubuntu 18.04.x    | Yes                 | No                 |
+| RedHat 7          | Yes                 | Yes                |
+| RedHat 6          | Yes                 | Yes                |
```

Comment:
Should be RedHat 8.

Reply:
Fixed and updated zoo.cfg with ansible_nodename.